JavaAssignments/src/amarja/Secondmax.java

```diff
@@ -0,0 +1,22 @@
+package amarja;
+//Question 5 Program to find second maximum number
+public class Secondmax {
+	int secmaximum(int num [])
+	{
+		int max=num[0];int secmax = 0;
+		for (int i=1;i<num.length;i++)
+		{
+			if (num[i]>max)
+			{	secmax=max;
+				max=num[i];
+			}
+		}
+		return secmax;	
+		}
+	public static void main(String[] args) {
+		Secondmax smax=new Secondmax();
+		Takeinputarrint input = new Takeinputarrint();//Accepting array input using method
+		int num[]=input.takeintegerinputuser();
+		System.out.println("Second maximum number is  "+smax.secmaximum(num));
```

Comment:
Your logic wont work for below input.
int num[]={10,14,99,110,108};

please check !

Reply:
Changed the logic and tested for other combinations as well